docker/dev/Dockerfile

```diff
@@ -121,6 +121,7 @@ RUN cd $GOPATH/src/github.com/ligato/vpp-agent \
 
 WORKDIR /
 
+RUN rm -f /dev/shm/db /dev/shm/global_vm /dev/shm/vpe-api
+
 # run supervisor as the default executable
-CMD rm -f /dev/shm/db /dev/shm/global_vm /dev/shm/vpe-api && \
-    /usr/bin/supervisord -c /etc/supervisord/supervisord.conf
+ENTRYPOINT ["/usr/bin/supervisord","-c","/etc/supervisord/supervisord.conf"]
```

Comment:
By using `ENTRYPOINT` instruction for starting supervisord you are forcing developers that want to run the container with shell only (no supervisord at all, perhaps for some debugging purposes) to override the entrypoint with `--entrypoint` flag instead of just appending custom command in the `docker run`.

`docker run -it ligato/vpp-agent /bin/bash` vs. `docker run -it --entrypoint /bin/bash ligato/vpp-agent`

Reply:
+1